pyproject.toml

```diff
@@ -52,6 +58,12 @@ cdk = [
     "aws-cdk.aws-iam",
     "aws-cdk.aws-dynamodb",
     "aws-cdk.aws-lambda",
+    "aws-cdk.aws-stepfunctions",
+    "aws-cdk.aws-stepfunctions_tasks",
+    "aws-cdk.aws-ecr",
+    "aws-cdk.aws-ecr_assets",
+    "aws-cdk.aws-ecs",
+    "aws-cdk.aws-ec2",
```

Comment:
We should sort these too, as [mentioned](https://github.com/linz/geospatial-data-lake/pull/56#discussion_r534509644).

Reply:
Should be fixed in 432e847b74c4d8c7184ffb3361a442696a726759